snap/info.go

```diff
@@ -249,10 +249,25 @@ func (app *AppInfo) WrapperPath() string {
 	return filepath.Join(dirs.SnapBinariesDir, binName)
 }
 
+func (app *AppInfo) invocation(command string) string {
+	securityTag := app.SecurityTag()
+	return fmt.Sprintf("/usr/bin/ubuntu-core-launcher %s %s %s", securityTag, securityTag, filepath.Join(app.Snap.MountDir(), command))
```

Comment:
Ah, interesting. What is the motivation behind this change? I don't disagree with it in any way, just curious.